src/core/services/HTTPClient.js

```diff
@@ -21,14 +20,11 @@ instance.interceptors.request.use(
 instance.interceptors.response.use(
   response => response.data,
   error => {
-    if (error.response.status === 401) {
+    if (error?.response?.status === 401) {
```

Comment:
why do you use `?` here?

Reply:
This short syntax is Optional chaining, it help you to check if object is null or not instead of ```error && error.response ? error.response.status : null```. You can read more at here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining